.babelrc.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-react',
+      {
+        runtime: 'automatic',
+        importSource: 'preact',
+      },
+    ],
+    [
+      '@babel/preset-env',
+      {
+        bugfixes: true,
+        modules: false,
```

Comment:
`modules: false` is the magic that results in ES module output in `lib/index.js`. See https://babeljs.io/docs/en/babel-preset-env#modules.

Reply:
Welp, we're not in JSON here, so you could add a helpful comment :)